app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+html xmlns="http://www.w3.org/1999/xhtml" 
+  head
+    = stylesheet_link_tag "email/email"
+    meta content="text/html; charset=utf-8" http-equiv="Content-Type"
+    meta content="width=device-width" name="viewport"
+    title
+      = message.subject
+  body
+    table.body data-made-with-foundation=""
+      tr
+        td.center align="center" valign="top" 
+          center data-parsed="" 
+            table.spacer.float-center
+              tbody
+                tr
+                  td height="10px" style="font-size:10px;line-height:10px;"   
+            table.container.float-center
+              tbody
+                tr
+                  td
+                    table.spacer
+                      tbody
+                        tr
+                          td height="35px" style="font-size:35px;line-height:35px;"   
+                    table.row
+                      tbody
+                        tr
+                          th.align-middle.small-12.large-12.columns.first.last
+                            table
+                              tr
+                                th
+                                  = image_tag("logo.png", class: "small-float-center")
```

Comment:
This will not render the image because it does not produce a full URL. To fix this, we need to use `asset_url` with a `host` parameter, such as:
```
= image_tag(asset_url("logo.png", host: Figaro.env.domain_name), class: "small-float-center")
```
The problem is that the `host` option needs a protocol as well, and our `domain_name` entries in `application.yml` do not include the protocol. This will need to be fixed in a separate PR, but in the meantime, you can create a new entry in `application.yml.example` called `mailer_domain_name` and set it to `http://localhost:3000` for development. Then you can use it for the image by calling `Figaro.env.mailer_domain_name`